mkosi/__init__.py

```diff
@@ -1640,28 +1640,28 @@ def reenable_kernel_install(args: CommandLineArguments, root: str) -> None:
     make_executable(hook_path)
 
 
-def make_rpm_list(args: argparse.Namespace, packages: List[str]) -> List[str]:
-    packages = list(packages)  # make a copy
+def make_rpm_list(args: argparse.Namespace, packages: Set[str]) -> Set[str]:
+    packages = set(packages)  # make a copy
```

Comment:
```suggestion
    packages = packages.copy()
```
I think this would be more explicit.